### DISCUSSION TITLE
my github account has been suspended ...

### DISCUSSION THREAD

**Discussion by fc6666666** [community/community#127530] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi CF,

my github account has been suspended suddenly, without any mail for the reason. I think i havenot abuse any service or violate tos....
And I've filed an appeal with Ticket ID: 2824779.
However no one replied after two days... Can any one help please ?

**Comment by loki135** [comment]
same here too, I'm waiting a week to get resolved still no response
